ionoscloud/resource_dbaas_mongo_user_test.go

```diff
@@ -213,11 +208,6 @@ resource ` + constant.DBaasMongoClusterResource + ` ` + constant.DBaaSClusterTes
     cidr_list            =  ["192.168.1.108/24", "192.168.1.109/24", "192.168.1.110/24"]
   }
   template_id = "6b78ea06-ee0e-4689-998c-fc9c46e781f6"
-  
-  credentials {
-  	username = "username"
-	password = ` + constant.RandomPassword + `.user_password.result
-  }
```

Comment:
i think we can remove  ``` resource ` + constant.RandomPassword + ` "user_password"``` from this config since it is no longer used

Reply:
@cbeti-ionos done in e6a85d03a5b14014f8b0a2c37f0cece844a069ba

Reply:
@cbeti-ionos done in fb35523b30cfd996201b649461669a4750c84e80